cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -324,43 +318,43 @@
     <property>
         <name>data.local.storage</name>
         <value>${local.data.dir}/ldb</value>
-        <description>Specifies the database directory</description>
+        <description>Database directory for data fabric when in standalone mode</description>
```

Comment:
Changed description to "Database directory for LevelDB, used for data fabric in standalone mode"